src/main/java/net/sf/jabref/gui/keyboard/KeyBindingCategory.java

```diff
@@ -23,7 +23,7 @@
 
 
     private KeyBindingCategory(String name) {
-        this.name = name;
+        this.name = name.replace("&", "");
```

Comment:
> The KeyBindingCategory is used as the header in the keybinding dialog and
thus has absolutely nothing to do with the menu.

Sure? I thought, this was intentional to mirror the menu structure here...
And thus reuse the menu translations?